test/e2e/util.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-openshift-controller-manager-operator/test/framework"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// getConfig helper to execute get against operator's config.
+func getConfig(t *testing.T, client *framework.Clientset) *operatorv1.OpenShiftControllerManager {
+	opts := metav1.GetOptions{}
+	cfg, err := client.OpenShiftControllerManagers().Get(context.TODO(), "cluster", opts)
+	if err != nil {
+		t.Fatalf("error getting openshift controller manager: '%v'", err)
+	}
+	return cfg
+}
+
+// updateConfig helper to execute update against operator's config.
+func updateConfig(
+	t *testing.T,
+	client *framework.Clientset,
+	cfg *operatorv1.OpenShiftControllerManager,
+) *operatorv1.OpenShiftControllerManager {
+	opts := metav1.UpdateOptions{}
+	updated, err := client.OpenShiftControllerManagers().Update(context.TODO(), cfg, opts)
+	if err != nil {
+		t.Fatalf("error updating openshift controller manager: '%v'", err)
+	}
+	return updated
+}
+
+// assertOperatorConditions compare two slices of operator conditions ignoring timestamps.
+func assertOperatorConditions(t *testing.T, expected, actual []operatorv1.OperatorCondition) bool {
+	for _, e := range expected {
+		matches := false
+		for _, a := range actual {
+			if matches {
+				continue
+			}
+			if e.Type == a.Type && e.Status == a.Status && e.Reason == a.Reason {
```

Comment:
Are there never any messages set on these conditions?

Reply:
Yes, there are messages in these conditions. It use to be snippet repeated around, and it has been refactored to a generic sub-routine.

Reply:
so are you saying you are validating the messages in another routine @otaviof ?  If so, can you put a url link to it here?

Reply:
https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/dbdac5e7ba188fcd0b35867cc7e9f6ce07e5c66e/pkg/operator/operator_test.go#L144-L159

Reply:
ah there you go ... thanks @otaviof